src/auro-flight.js

```diff
@@ -34,16 +34,10 @@ import "./auro-flight-main";
 
 // build the component class
 class AuroFlight extends LitElement {
-
-  constructor() {
-    super();
-
-    this.ariaHidden = false;
-  }
-
   // function to define props used within the scope of this component
   static get properties() {
     return {
+      stops: { type: Array },
```

Comment:
With the addition of this attribute, this needs to be documented in the JSdoc descriptions.

Reply:
Fixed